nrpy/c_codegen.py

```diff
@@ -32,14 +33,30 @@
     "float",
     "long double",
     # Standard C++ types
-    "std::float16_t",
     "std::float32_t",
     "std::float64_t",
-    "std::float128_t",
-    "std::bfloat16_t",
-    # Maybe add complex types?
+    # Unsupported types by sympy ccode generator
+    # "std::bfloat16_t",
+    # "std::float16_t",
+    # "__float128",
+    # "std::float128_t",
 ]
 
+fp_type_to_sympy_type = {
+    # Traditional C types
+    "double": sp_ast.float64,
+    "float": sp_ast.float32,
+    "long double": sp_ast.float80,
+    # Standard C++ types
+    "std::float32_t": sp_ast.float32,
+    "std::float64_t": sp_ast.float64,
+    # Unsupported types by sympy ccode generator
```

Comment:
Let's remove this dead code -- no need to keep it around.

Reply:
Resolved in cd817074a1c5cb5c5c444242a2d2a368da6391dc